src/platform_impl/windows/window.rs

```diff
@@ -795,18 +801,18 @@ where
     };
 
     let (style, ex_style) = window_flags.to_window_styles();
-    let handle = winuser::CreateWindowExW(
+    let handle = CreateWindowExW(
         ex_style,
-        class_name.as_ptr(),
-        title.as_ptr() as LPCWSTR,
+        PWSTR(class_name.as_mut_ptr()),
+        OsStr::new(&attributes.title),
```

Comment:
```suggestion
        attributes.title,
```

I hope we can get rid of the `PWSTR` pointer above, too.

Reply:
This is not possible due to disabled `std` feature